src/test/deploy/functions/validate.spec.ts

```diff
@@ -198,6 +199,101 @@ describe("validate", () => {
         /they have fewer than 2GB memory/
       );
     });
+
+    it("Disallows multiple beforeCreate blocking", () => {
```

Comment:
nit - prefer lowercase `disallow` (same comment for tests below).

Reply:
fixed in #4443